src/api/config/routes.rb

```diff
@@ -31,7 +31,7 @@ def self.public_or_about_path?(request)
     arch:       %r{[^\/]*},
     binary:     %r{[^\/]*},
     filename:   %r{[^\/]*},
-    id:         %r{\d*},
+    id:         %r{[^\/]*},
```

Comment:
This is everything except `/` and the previous one everything, but `/` is used a separator in the url, so this is equivalent anyway.